src/main/java/ch/uzh/soprafs22/groupmatcher/controller/MatcherController.java

```diff
@@ -7,6 +7,7 @@
 import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.GetMapping;
```

Comment:
Same about being kinda silly but it would be great if you could remove this extra line (it's not needed since we have the * import on line 9) because then MatcherController.java wouldn't have any diffs / wouldn't be in this PR, which will prevent any potential hassle when merging the other PRs